fearless/Modules/Profile/ProfilePresenter.swift

```diff
@@ -54,7 +54,10 @@ extension ProfilePresenter: ProfilePresenterProtocol {
         guard let wallet = selectedWallet else {
             return
         }
-        wireframe.showAccountDetails(from: view, metaAccount: wallet)
+        let walletNameChangeCompletion: (MetaAccountModel) -> Void = { [weak self] wallet in
```

Comment:
We have already begun a discussion that should be continued. I find the use of "*ModuleOutput" protocols much more convenient

Reply:
This is modal screen, which can be opened from three different places. Completion seems the best option for me in that case.